src/assets/dotnet.md

```diff
@@ -1,5 +1,54 @@
 ## .NET
 ---
 ### Prerequisites
+In order to use Allure in your .Net project with the following exemple you will need used NUnit with NuGet NUnit.Allure
 
 ### Writing a Test case
+The tag [AllureNUnit] its necessary to generate the JSON files.
```

Comment:
I suggest 'The [AllureNUnit] tag it is necessary to generate the JSON files'.

Reply:
changed